hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/JpaAction.java

```diff
@@ -137,6 +137,9 @@
     @Column(name = "external_ref", length = Action.EXTERNAL_REF_MAX_LENGTH)
     private String externalRef;
 
+    @Column(name = "triggered_by", updatable = false, nullable = false, length = 64)
```

Comment:
Shouldn't this rather be `updatable = true`?

Reply:
no the action should have only 1 triggerer, why should this value be updatable?

Reply:
Indeed, you're right. The action won't be changed after its creation